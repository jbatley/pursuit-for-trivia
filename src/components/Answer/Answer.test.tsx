import React from 'react';
import { render } from '@testing-library/react';
import Component from './Answer';

describe('Answer component', () => {
  test('matches snapshot', async () => {
    const { container } = render(<Component text="123" id="1" onChange={() => {}} selected="123" />);
    expect(container.firstChild).toMatchSnapshot();
  });
  test('matches snapshot', async () => {
    const { container } = render(<Component text="123" id="1" onChange={() => {}} selected="1234" />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
