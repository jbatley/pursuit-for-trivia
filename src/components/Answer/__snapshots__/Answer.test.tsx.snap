// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Answer component matches snapshot 1`] = `
<div
  class="container"
>
  <input
    checked=""
    class="checkbox"
    id="button-1"
    name="answer"
    type="radio"
    value="123"
  />
  <label
    class="label"
    for="button-1"
  >
    123
    <svg
      class="icon"
    >
      checkmark.svg
    </svg>
  </label>
</div>
`;

exports[`Answer component matches snapshot 2`] = `
<div
  class="container"
>
  <input
    class="checkbox"
    id="button-1"
    name="answer"
    type="radio"
    value="123"
  />
  <label
    class="label"
    for="button-1"
  >
    123
    <svg
      class="icon"
    >
      checkmark.svg
    </svg>
  </label>
</div>
`;
